modules/apps/youtube/youtube-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -25,11 +25,74 @@
 				</aui:a>
 			</c:when>
 			<c:otherwise>
-				<iframe allowfullscreen frameborder="0" height="<%= youTubeDisplayContext.getHeight() %>" src="<%= youTubeDisplayContext.getEmbedURL() %>" width="<%= youTubeDisplayContext.getWidth() %>" wmode="Opaque" /></iframe>
+				<iframe allowfullscreen frameborder="0" id="<portlet:namespace />iframe" src="<%= youTubeDisplayContext.getEmbedURL() %>" wmode="Opaque" /></iframe>
 			</c:otherwise>
 		</c:choose>
 	</c:when>
 	<c:otherwise>
 		<liferay-util:include page="/html/portal/portlet_not_setup.jsp" />
 	</c:otherwise>
-</c:choose>
\ No newline at end of file
+</c:choose>
+
+<aui:script>
+	function <portlet:namespace />resizeIFrame() {
+		var iframe = document.getElementById('<portlet:namespace />iframe');
+
+		if (iframe != null) {
+			var displayContextHeight = <%= youTubeDisplayContext.getHeight() %>;
+			var displayContextWidth = <%= youTubeDisplayContext.getWidth() %>;
+
+			var parent = iframe.parentElement;
+			var parentWidth = parent.offsetWidth;
+
+			if (displayContextWidth > parentWidth) {
+				displayContextWidth = parentWidth;
+			}
+
+			iframe.setAttribute('height', displayContextHeight);
+			iframe.setAttribute('width', displayContextWidth);
+		}
+	};
+
+	function <portlet:namespace />addDragAndDropListener() {
+		if (!Liferay.Layout) {
+			setTimeout(
+				function() {
+					<portlet:namespace />addDragAndDropListener();
+				},
+				10
+			);
+		}
+		else {
+			Liferay.Layout.on(
+				[ 'drag:end', 'drag:start' ],
```

Comment:
Hey @ChrisKian @jrao @SpencerWoo,

The `Layout` drag events are not firing with this change.  Can you double check and make sure this is doing what you intend?

![bug](https://user-images.githubusercontent.com/860951/53443713-255f8880-39c1-11e9-877a-73f2c0eb57ee.gif)

Reply:
@jonmak08 I'm not able to reproduce the issue shown by your gif. Tried in Chrome and Firefox.

I did run into that issue during development but since I can no longer reproduce it, I thought it was solved. Could the issue be due to a cached JS file? Maybe try again in incognito mode?

Reply:
Hey @jonmak08, I also tested and confirmed I cannot reproduce the issue you are seeing.

Reply:
@jonmak08 I asked @ChrisKian to test this as well. He's also not able to reproduce the issue in your gif. Maybe you could test it again?